test/compiler.test.ts

```diff
@@ -3,8 +3,7 @@ import { tmpdir } from 'node:os';
 import { dirname, join } from 'node:path';
 import { loadAssemblyFromPath, SPEC_FILE_NAME, SPEC_FILE_NAME_COMPRESSED } from '@jsii/spec';
 import { compile, Lock } from './fixtures';
-import { Compiler } from '../src/compiler';
-import { TYPES_COMPAT } from '../src/downlevel-dts';
+import { Compiler, TYPES_COMPAT } from '../src/compiler';
```

Comment:
```suggestion
import { Compiler } from '../src/compiler';
```

Reply:
Please adjust the tests in this file (and possibly code)
`TYPES_COMPAT` should not be emitted anymore.